src/Cake.Common/Tools/OpenCover/OpenCoverHideSkippedOption.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Cake.Common.Tools.OpenCover
+{
+    using System;
+
+    /// <summary>
+    /// Represents the hide skipped option for OpenCover.
+    /// </summary>
+    [Flags]
+    public enum OpenCoverHideSkippedOption
+    {
+        /// <summary>
+        /// Removes information that skipped due to switch <value>-excludebyfile </value> from output file.
+        /// </summary>
+        File = 1,
+
+        /// <summary>
+        /// Removes information that skipped due to switch <value>-filter </value> from output file.
+        /// </summary>
+        Filter = 2,
+
+        /// <summary>
+        /// Removes information that skipped due to switch <value>-excludebyattribute </value> from output file.
+        /// </summary>
+        Attribute = 4,
+
+        /// <summary>
+        /// Removes missing Pdb information from output file.
+        /// </summary>
+        MissingPdb = 8,
+
+        /// <summary>
+        /// Removes all information from output file.
+        /// </summary>
+        All = 16
```

Comment:
Shouldn't all be the combination of all of them? So like:
` All = File | Filter | Attribute | MissingPdb`

Reply:
If I add like above then when it gets translated to string, it'll look like `-hideskipped:File;Filter;Attribute;MissingPdb` however it can take input as `-hideskipped:All`

Reply:
You could check in the runner if it is equal all and then replace by all. This would also replace it to All in case someone defines all individual flags.

Reply:
I get it but I see that it also supports this `File;Filter;Attribute;MissingPdb;All`, do we need to append other options if All is appended already? If that's redundant then I can make the change.

Reply:
@Roemer I've updated the code as per the suggestion, Also added few more unit tests to cover `All` scenario, please check.